batch.go

```diff
@@ -135,7 +135,7 @@ func (b *Batch) Send(ctx context.Context, txOptions *TxOptions) error {
 
 	_, err = b.conn.conn.Write(buf)
 	if err != nil {
-		b.conn.die(err)
+		b.die(err)
```

Comment:
I believe every other place in this file, we use `b.die` and I wasn't sure why this case would need to be different.